src/helpers/helpers.canvas.js

```diff
@@ -32,8 +32,10 @@ export function clear(chart) {
 	chart.ctx.clearRect(0, 0, chart.width, chart.height);
 }
 
-export function drawPoint(ctx, style, radius, x, y, rotation) {
+export function drawPoint(ctx, options, radius, x, y) {
 	var type, xOffset, yOffset, size, cornerRadius;
+	var style = options.pointStyle;
+	var rotation = options.rotation;
```

Comment:
can take `radius` from options too, getting down to the CC acceptable 4 parameters

also caching of `radius` in `Point.draw` is not needed after that, so can remove line 67 and use `options.radius` in line 69. (in elements.point.js)

Reply:
Thanks @kurkle ! I see you already approved the request - does that mean you already implemented your suggested changes above, or should I make them?